docs/userguide/parsec_workspaces.rst

```diff
@@ -138,3 +138,52 @@ workspace.
   Strong cryptographic security prevents data recovery. If the workspace is
   not shared with others, and the user loses access to its device or cannot
   log in for any reason, data stored in the workspace will be lost forever.
+
+See a workspace history
+-----------------------
+
+Parsec lets you explore a workspace at a given point in time, showing you all the files as they used to be.
+You will need the **Owner** or **Manager** roles on the workspace.
+The ``History`` option is available in the workspace context menu.
+
+.. image:: screens/workspace_context_menu_history.png
+    :align: center
+    :width: 250
+    :alt: See workspace history
+
+You can navigate inside the workspace as you normally would.
+
+.. image:: screens/workspace_history.png
+    :align: center
+    :alt: Workspace history
+
+If you change the date or time, files and folders will be automatically updated to reflect the state of the workspace at this moment.
+You can only select a time between the workspace's creation date and now.
+
+.. image:: screens/workspace_history_select_date.png
+    :align: center
+    :width: 300
+    :alt: Select a date and time
+
+If you wish to restore a file or a folder, you can select it and click ``Restore``. This will replace the current version of the file with the version from this moment in time.
```

Comment:
```suggestion
If you want to restore a file or a folder, select it and click ``Restore``. This will replace the current version of the file with the version from the selected date and time.
```